test/conv_common.hpp

```diff
@@ -178,11 +179,18 @@ struct conv_stats
 template <class T, class Tout = T>
 tensor<Tout> get_output_tensor(const miopen::ConvolutionDescriptor& filter,
                                const tensor<T>& input,
-                               const tensor<T>& weights)
+                               const tensor<T>& weights,
+                               std::string out_layout = "")
```

Comment:
As far as I see, the default is not used.
```suggestion
                               const tensor<T>& weights,
                               const std::string& out_layout)
```

Reply:
Indeed this function is used in multiple files like in [cba_inference.cpp](https://github.com/ROCmSoftwarePlatform/MIOpen/blob/develop/test/cba_inference.cpp#L86). since fusion kernel currently have no kernel support different layout, which lead to no suitable test or validation, so I just leave it cba/cbna unchanged and give this function a default value of `out_layout`

Reply:
No, that files use definition from `fusionHost.hpp`.

Reply:
This is not resolved.

Reply:
OK, fixed in https://github.com/ROCmSoftwarePlatform/MIOpen/pull/765/commits/ee0302f0bd4ffe149ab55fbb80437773ab7cf148